onnx/external_data_helper.py

```diff
@@ -112,22 +114,32 @@ def convert_model_to_external_data(model, all_tensors_to_one_file=True, location
               If not specified, will use the model name.
     size_threshold: Threshold for size of data. Only when tensor's data is >= the size_threshold
     it will be converted to external data. To convert every tensor with raw data to external data set size_threshold=0.
+    convert_attribute: If true, convert all tensors to external data
+                       If false, convert only non-attribute tensors to external data
+    model_file_path: A string containing a file name
     """
+    tensors = _get_initializer_tensors(model)
+    if convert_attribute:
+        tensors = _get_all_tensors(model)
+
     if all_tensors_to_one_file:
         file_name = Text(uuid.uuid1())
         if location:
             file_name = location
-        for tensor in _get_all_tensors(model):
+        for tensor in tensors:
             if tensor.HasField("raw_data") and sys.getsizeof(tensor.raw_data) >= size_threshold:
                 set_external_data(tensor, file_name)
     else:
-        for tensor in _get_all_tensors(model):
+        for tensor in tensors:
             if tensor.HasField("raw_data") and sys.getsizeof(tensor.raw_data) >= size_threshold:
                 tensor_location = tensor.name
                 if not _is_valid_filename(tensor_location):
                     tensor_location = Text(uuid.uuid1())
                 set_external_data(tensor, tensor_location)
 
+    if model_file_path:
```

Comment:
I didn't make model_file_path a requirement to make sure it doesn't break the behavior of end-user

Reply:
cc @askhade

Reply:
thinking about it more, it makes sense to remove this and make model_file_path a requirement. Why convert if not save?

Reply:
I ended up removing this check and making the file path required parameter. Let me know if this works and if we want to go back to making it optional.

Reply:
After speaking with Ashwini, we decided to move the model_file_path out of the existing API to remove the requirements for the model file path.